taxonomy-editor-frontend/package.json

```diff
@@ -7,6 +7,7 @@
     "@emotion/styled": "^11.9.3",
     "@material-table/core": "^0.2.35",
     "@mui/icons-material": "^5.8.4",
+    "@mui/lab": "^5.0.0-alpha.103",
```

Comment:
from what I could see the only thing we are using from it is a loading button, maybe we could show a loading spinner (we already have the icons package). Don't see it adds much value for the moment, and would be another dependecy

Reply:
Ok, I removed the dependency.